kolibri/plugins/learn/assets/src/views/HybridLearningContentCardListView.vue

```diff
@@ -158,6 +175,51 @@
         return this.coreString('bookmarkedTimeAgoLabel', { time });
       },
     },
+    methods: {
+      categoryAndLevelString(content) {
+        if (this.levels(content.grade_levels) && this.category(content.categories)) {
+          return this.category(content.categories) + ' | ' + this.levels(content.grade_levels);
+        } else if (this.category(content.categories)) {
+          return this.category(content.categories);
+        } else if (this.levels(content.grade_levels)) {
+          return this.levels(content.grade_levels);
+        }
+        return null;
+      },
+      levels(levels) {
+        const levelsNonReactive = JSON.parse(JSON.stringify(levels));
```

Comment:
What was it about the levels array that was causing this to be an issue?

Reply:
it was reactive and I was very sure it was the problem, but I've just removed this and it's rendering fine, so clearly it was something else that was causing it not to render!